### DISCUSSION TITLE
[A-L] (20 Giugno 2024) Esercizio 4

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#414] [Esami A-L]
Dimostrare per induzione che per ogni numero naturale $\Large n \geq 1$ esiste un $\Large k \geq 1$ tale che $\Large 2^{2n} − 1 = 3 \cdot k$.

Individuare il Caso Base, l’Ipotesi Induttiva e il Passo Induttivo.

**Comment by cek753** [comment]
![image](https://github.com/user-attachments/assets/2cccbccf-d4ab-4bd9-8948-cb215a5f40e2)
